official/resnet/keras/keras_imagenet_main.py

```diff
@@ -116,17 +116,20 @@ def run(flags_obj):
     distribution_utils.undo_set_up_synthetic_data()
     input_fn = imagenet_main.input_fn
 
-  train_input_dataset = input_fn(is_training=True,
-                                 data_dir=flags_obj.data_dir,
-                                 batch_size=flags_obj.batch_size,
-                                 num_epochs=flags_obj.train_epochs,
-                                 parse_record_fn=parse_record_keras)
-
-  eval_input_dataset = input_fn(is_training=False,
-                                data_dir=flags_obj.data_dir,
-                                batch_size=flags_obj.batch_size,
-                                num_epochs=flags_obj.train_epochs,
-                                parse_record_fn=parse_record_keras)
+  train_input_dataset = input_fn(
+      is_training=True,
+      data_dir=flags_obj.data_dir,
+      batch_size=flags_obj.batch_size,
+      num_epochs=flags_obj.train_epochs,
+      parse_record_fn=parse_record_keras,
+      datasets_num_private_threads=flags_obj.datasets_num_private_threads)
+
+  eval_input_dataset = input_fn(
+      is_training=False,
+      data_dir=flags_obj.data_dir,
+      batch_size=flags_obj.batch_size,
+      num_epochs=flags_obj.train_epochs,
+      parse_record_fn=parse_record_keras)
```

Comment:
what do you think about passing to the eval dataset also?

Reply:
I am not very clear about the impact of doing so. Would that have any impact on training dataset? Before figuring this out, I would err on the side of not doing so.